sparse/rest/handlers.go

```diff
@@ -22,6 +22,23 @@ type SyncFileStub struct{}
 
 func (f *SyncFileStub) UpdateSyncFileProgress(size int64) {}
 
+func (server *SyncServer) getQueryDirectIO(request *http.Request) (bool, error) {
+	var directIO bool
+	var err error
```

Comment:
redundant if you modify line 35 to the following: 
```
directIO, err := strconv.ParseBool(directIOStr)
```

Reply:
updated